tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/server/AutoCreateExpiryResourcesTest.java

```diff
@@ -194,6 +195,8 @@ public void testWithJMSFQQN() throws Exception {
 
       triggerExpiration();
 
+      Thread.sleep(EXPIRY_DELAY);
+
```

Comment:
Is this actually needed? If so maybe a comment why given the other tests dont do it? Given the triggerExpiration method before seems to send an already-expired message, and the later receive here already has a timeout.

Reply:
I actually added a comment why